src/main/java/org/example/MercedesCKlasse.java

```diff
@@ -0,0 +1,53 @@
+package org.example;
+
+public class MercedesCKlasse extends Mercedes {
+    public MercedesCKlasse(float fuelTankSize, int gears, String fuelType, int tireSize, String chassisNumber) {
+        super(fuelTankSize, gears, fuelType, tireSize, chassisNumber);
+    }
+
+    /**
+     * Method starts the engine and sets the initial fuel consumption based on tyre size.
+     */
+    @Override
+    public void start() {
+        super.start();
+        System.out.println("The Mercedes CKlasse started.");
```

Comment:
we already have an algorithm for fuel, why duplicate the code? instead we can make the start method use our specific consumptionPer100Km

Reply:
still duplicated code here